linkchecker

```diff
@@ -204,53 +245,81 @@ def has_encoding(encoding):
     except LookupError:
         return False
 
+
 # instantiate option parser and configure options
 argparser = LCArgumentParser(
-  epilog=Epilog,
-  formatter_class=argparse.RawDescriptionHelpFormatter
+    epilog=Epilog, formatter_class=argparse.RawDescriptionHelpFormatter
 )
 
 
 # build a config object for this check session
 config = linkcheck.configuration.Configuration()
 config.set_status_logger(console.StatusLogger())
 
-################# general options ##################
+# ================== general options =====================
 group = argparser.add_argument_group(_("General options"))
-group.add_argument("-f", "--config", dest="configfile",
-                 metavar="FILENAME",
-                 help=_(
-"""Use FILENAME as configuration file. Per default LinkChecker uses
+group.add_argument(
+    "-f",
+    "--config",
+    dest="configfile",
+    metavar="FILENAME",
+    help=_(
+        """Use FILENAME as configuration file. Per default LinkChecker uses
 ~/.linkchecker/linkcheckerrc (under Windows
-%%HOMEPATH%%\\.linkchecker\\linkcheckerrc)."""))
-group.add_argument("-t", "--threads", type=int, metavar="NUMBER",
-                 help=_(
-"""Generate no more than the given number of threads. Default number
-of threads is 10. To disable threading specify a non-positive number."""))
-group.add_argument("-V", "--version", action="store_true",
-                 help=_("""Print version and exit."""))
-group.add_argument("--list-plugins", action="store_true", dest="listplugins",
-                 help=_(
-"""Print available check plugins and exit."""))
-group.add_argument("--stdin", action="store_true",
-                 help=_(
-"""Read list of white-space separated URLs to check from stdin."""))
-
-################# output options ##################
+%%HOMEPATH%%\\.linkchecker\\linkcheckerrc)."""
+    ),
+)
+group.add_argument(
+    "-t",
+    "--threads",
+    type=int,
+    metavar="NUMBER",
+    help=_(
+        """Generate no more than the given number of threads. Default number
+of threads is 10. To disable threading specify a non-positive number."""
```

Comment:
I strongly want to convert these tripple-quoted strings into regular concatenated string literals like this

```suggestion
        "Generate no more than the given number of threads. Default number\n"
        "of threads is 10. To disable threading specify a non-positive number."
```

because the way they break the flow of indentation makes reading this code hard.

Alternatively we could use textwrap.dedent(), but when the strings are 2 or 3 lines I think that's overkill.

Reply:
Also, these are inside _(), so textwrap.dedent() would mess up translation extraction, I'm sure.

Reply:
I've done those - the only triples remaining are in the globals.